src/app/profile/yourGiving/stopStartRecurringGifts/restartGift/step0/addUpdatePaymentMethod.component.js

```diff
@@ -0,0 +1,86 @@
+import angular from 'angular';
```

Comment:
Did you make any changes to this component? It looks mostly copied and pasted... I didn't look too closely. Idk how reusable I made it. It could be a bit of effort.